### PR TITLE
Make single page app

### PR DESCRIPTION
**Pull request by aattsai**

### CODE REVIEW

File: app/views/partials/menuItems.html [hunk 1]
```diff
@@ -33,10 +32,22 @@ <h1 class="restaurant-name">{{restaurant.name}}</h1>
         <p>Section: {{section.name}}</p>
         <hr>
         <li ng-repeat="item in section.items | matchClickedItems:tagFilters">
-
-          <a ng-href="/restaurants/{{restaurantId}}/menuItems/{{item.id}}">
-            <p>Item: {{item.name}}</p>
-            <p>Price:  {{ item.price | currency : $ : 2 }}</p>
+          <a ng-href="#/restaurants/{{restaurantId}}/menuItems/{{item.id}}">
```
Comment (Yoonis) [inline app/views/partials/menuItems.html:37]: Place <a> tags on Item and Price only, exclude the tags.

File: app/views/partials/menuItems.html [hunk 2]
```diff
@@ -33,10 +32,22 @@ <h1 class="restaurant-name">{{restaurant.name}}</h1>
...
+          <p>Item: {{item.name}}</p>
+          <p>Description: {{item.description}}</p>
+          <p>Price:  {{ item.price | currency : $ : 2 }}</p>
+          <label>Tags: </label>
+          <ul>
```
Comment (Yoonis) [inline app/views/partials/menuItems.html]: Change back from ul and li to p and span tags.

**Comment by tinglycraniumplacidly** [comment]
:+1:

File: app/views/partials/menuItems.html [hunk 3]
```diff
@@ -61,3 +61,4 @@ <h1 class="restaurant-name">{{restaurant.name}}</h1>
     </div>
   </ul>
 </div>
+Status API Training Shop Blog About Pricing
```
Comment (Yoonis) [inline app/views/partials/menuItems.html]: Jon says remove this!

**Comment by Yoonis** [comment]
:+1:

**Pull request merged**